### PR TITLE
Send locust report with ci helper

### PR DESCRIPTION
**Pull request by kompotkot**
Added possibility to send locust reports to your servers.

### CODE REVIEW

**Comment by kompotkot** [comment]
Instead of send call it publish

File: locust/ci_helpers/github.py [hunk 1]
```diff
@@ -19,33 +41,93 @@ def generate_argument_parser() -> argparse.ArgumentParser:
...
+def publish(
+    repo_url: str,
+    initial: str,
+    terminal: str,
+    comments_url: str,
+) -> str:
+    """
+    Publish locust summary to API.
+    """
+    git_result = git.run(".", initial, terminal)
```
Comment (zomglings) [inline locust/ci_helpers/github.py:53]: Take plugins arg from command line.

Comment (zomglings) [reply locust/ci_helpers/github.py:53]: See https://github.com/bugout-dev/locust/blob/main/locust/cli.py#L41

**Review by zomglings — APPROVED** [review]
lgtm, please publish a fix in a separate PR with plugins argument to `locust.git publish` - you can use the right `populate_argument_parser` function from locust.

**Pull request merged**